sys-fs/squashfs_tools/squashfs_tools-4.3.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Provides tools for creating and manipulating squashfs filesystems"
+DESCRIPTION="squashfs-tools provides tools for creating and appending to \
+(mksquashfs) and extracting (unsquashfs) squashfs filesystems.
+
+Squashfs is a compressed read-only filesystem. It is intended for general \
+read-only filesystem use, archival use (i.e. in cases where a .tar.gz file \
+may be used, and in constrained block device/memory systems (e.g. embedded \
+systems) where low overhead is needed."
+HOMEPAGE="http://squashfs.sourceforge.net"
+COPYRIGHT="2002-2014 Philip Lougher"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://prdownloads.sf.net/squashfs/squashfs$portVersion.tar.gz"
+CHECKSUM_SHA256="0d605512437b1eb800b4736791559295ee5f60177e102e4d4ccd0ee241a5f3f6"
+SOURCE_DIR="squashfs$portVersion/squashfs-tools"
+PATCHES="
+	squashfs_tools-$portVersion-2gb.patch
+	squashfs_tools-$portVersion-aligned-data.patch
+	squashfs_tools-$portVersion-cve2015-4645_4646.patch
+	squashfs_tools-$portVersion-haiku-compat.patch
+	squashfs_tools-$portVersion-fix-mem-calculation.patch
+	squashfs_tools-$portVersion-local-cve-fix.patch
+	squashfs_tools-$portVersion-lzma-magic.patch
+	squashfs_tools-$portVersion-mem-overflow.patch
+	squashfs_tools-$portVersion-xattrs.patch
+	"
+
+DEBIAN_REV="3"
+SOURCE_URI_2="http://http.debian.net/debian/pool/main/s/squashfs-tools/squashfs-tools_$portVersion-$DEBIAN_REV.debian.tar.xz"
```

Comment:
It would be slightly better to use the canonical name:(httpredir.debian.org instead of http.debian.net):
```
SOURCE_URI_2="http://httpredir.debian.org/debian/pool/main/s/squashfs-tools/squashfs-tools_$portVersion-$DEBIAN_REV.debian.tar.xz"
```